engines/dradis-api/spec/requests/dradis/ce/api/v1/evidence_spec.rb

```diff
@@ -82,13 +82,13 @@
         ev_1 = retrieved_evidence.find { |n| n["issue"]["id"] == @issues[1].id }
         ev_2 = retrieved_evidence.find { |n| n["issue"]["id"] == @issues[2].id }
 
-        expect(ev_0["fields"].keys).to match_array %w[Label a]
+        expect(ev_0["fields"].keys).to match_array %w[Label Title a]
```

Comment:
since `#local_fields` is public, can we `match_array` against `ev_0.local_fields + a` to future-proof this spec?